pkgs/applications/video/aegisub/default.nix

```
@@ -113,6 +127,10 @@ stdenv.mkDerivation rec {
     "relro"
   ];
 
+  postPatch = ''
+    sed -i 's/-Wno-c++11-narrowing/-Wno-narrowing/' {configure.ac,src/Makefile}
```

Why not the following?

```suggestion
    sed -i 's/-Wno-c++11-narrowing/-Wno-narrowing/' configure.ac src/Makefile
```